qiskit_machine_learning/connectors/__init__.py

```diff
@@ -0,0 +1,35 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Connectors (:mod:`qiskit_machine_learning.connectors`)
+===============================================================
+
+Connectors from Qiskit Machine Learning to other frameworks.
+
+.. currentmodule:: qiskit_machine_learning.connectors
+
+Connectors
+======================
+
+.. autosummary::
+   :toctree: ../stubs/
+   :nosignatures:
+
+   TorchConnector
+
+"""
+
+from .torch_connector import TorchConnector
+
+
+__all__ = ["TorchConnector"]
```

Comment:
See my comment below in terms of conditional import. If there is more than one connector from different frameworks here then we need to be careful that using the one you want does not fail because some other connector is loaded and its dependent pkg is not installed.